### ISSUE TITLE
DNS not working on some resolvers

### ISSUE THREAD

**Issue by iojcde** [packetframe/legacy#21]
I can't reach any of the AS34553 websites with cloudflare's 1.1.1.1 DNS, or with my unbound recursive dns resolver.
```
$ dig natesales.net @1.1.1.1

; <<>> DiG 9.16.11 <<>> natesales.net @1.1.1.1
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 61194
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
; EDE: 22 (No Reachable Authority)
;; QUESTION SECTION:
;natesales.net.                 IN      A

;; Query time: 50 msec
;; SERVER: 1.1.1.1#53(1.1.1.1)
;; WHEN: Fri Jul 09 19:18:48 KST 2021
;; MSG SIZE  rcvd: 48
```
`dig` complains about `No reachable authority`.

192.168.219.110 is my Pi-hole instance using unbound as the resolver.
```
$ dig natesales.net @192.168.219.110 

; <<>> DiG 9.16.11 <<>> natesales.net @192.168.219.110
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: SERVFAIL, id: 18064
;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1472
;; QUESTION SECTION:
;natesales.net.                 IN      A

;; Query time: 30 msec
;; SERVER: 192.168.219.110#53(192.168.219.110)
;; WHEN: Fri Jul 09 19:24:23 KST 2021
;; MSG SIZE  rcvd: 42

```
However, some resolvers like google's 8.8.8.8 works. This might be related to DNS caching, though. 
```
$ dig natesales.net @8.8.8.8

; <<>> DiG 9.16.11 <<>> natesales.net @8.8.8.8
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 63170
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 512
;; QUESTION SECTION:
;natesales.net.                 IN      A

;; ANSWER SECTION:
natesales.net.          21290   IN      A       23.141.112.33

;; Query time: 80 msec
;; SERVER: 8.8.8.8#53(8.8.8.8)
;; WHEN: Fri Jul 09 19:23:48 KST 2021
;; MSG SIZE  rcvd: 58

```

Other websites such as `packetframe.com`, `fosshost.org`'s DNS are not working either, but I expect that they are all hosted from the same infrastructure.

**Comment by natesales** [comment]
Can I see the output of `traceroute ns1.packetframe.com` ?

**Comment by iojcde** [comment]
Actually, traceroute is also failing since it can't resolve the IP address. 

Here's traceroute on the IP resolved from 8.8.8.8:

```
[jcde@fridge:~]$ traceroute 66.248.235.2                    (07-10 06:24)
traceroute to 66.248.235.2 (66.248.235.2), 30 hops max, 60 byte packets
 1  _gateway (192.168.219.1)  0.825 ms  0.795 ms  1.138 ms
 2  REDACTED (REDACTED)  2.523 ms  2.206 ms  3.022 ms
 3  10.240.248.93 (10.240.248.93)  6.295 ms  4.481 ms  3.616 ms
 4  1.208.9.129 (1.208.9.129)  2.464 ms  2.708 ms 1.208.9.165 (1.208.9.165)  2.696 ms
 5  1.208.9.165 (1.208.9.165)  3.550 ms  3.895 ms 1.208.9.129 (1.208.9.129)  3.523 ms
 6  1.213.145.13 (1.213.145.13)  4.152 ms  4.306 ms 1.213.147.5 (1.213.147.5)  2.784 ms
 7  1.213.152.193 (1.213.152.193)  3.031 ms  3.373 ms 1.208.167.141 (1.208.167.141)  3.188 ms
 8  1.208.104.57 (1.208.104.57)  1.900 ms 1.213.153.9 (1.213.153.9)  3.103 ms 203.252.10.209 (203.252.10.209)  2.608 ms
 9  1.208.104.222 (1.208.104.222)  39.249 ms 203.252.13.22 (203.252.13.22)  37.671 ms 203.252.13.18 (203.252.13.18)  37.380 ms
10  1.213.148.154 (1.213.148.154)  40.371 ms  37.328 ms 1.208.150.34 (1.208.150.34)  41.248 ms
11  * * *
12  100ge10-1.core1.tyo1.he.net (184.105.64.130)  160.831 ms  157.486 ms  157.468 ms
13  10ge11-11.core1.hnl1.he.net (184.105.213.85)  176.240 ms  174.977 ms  175.670 ms
14  184.105.247.178 (184.105.247.178)  171.561 ms  174.676 ms  175.809 ms
15  * * *
```
(REDACTED is just my IP address)

**Comment by iojcde** [comment]
Seems to be working now for some reason..

Closing this issue.
